GitKraken tutorial complete

suttonwilliamd opened the pull request:
I only speak English, so "Tutorials Using Other Tools" in every translation are Google Translated.

Roshanjossey commented:
@suttonwilliamd: Do you wanna squash these commits?

I remember that was one of the learnings we expected from this.

It's okay if you wanna take it up in a different PR too

suttonwilliamd commented:
I'd love to.  I still don't know how though.  I assumed that what I did wrong with that other project was submitting a pull request to myself and accepting it.

Roshanjossey commented:
Here's a nice material on how to squash commits.

https://www.devroom.io/2011/07/05/git-squash-your-latests-commits-into-one/

If you have any questions, do ask me. I'll be available in slack team as well

suttonwilliamd committed:
GitKraken tutorial complete

I only speak English, so "Tutorials Using Other Tools" in every translation are Google Translated.

suttonwilliamd commented:
Yes!  I did it!  I'm a genius!   ... Now accept it before I break it again.  haha.  Thanks for the link, Roshanjossey.

Roshanjossey commented:
@suttonwilliamd That's awesome. I'm so happy for you mate.

Also we have to re-open your PR in Kramdown

Merging
